src/emm/emm_cli.py

```diff
@@ -274,5 +278,15 @@ def git_commit(ctx: click.Context, commit_message: str) -> None:
     orchestrator.git_commit_modules(commit_message)
 
 
+@cli.command(
+    context_settings=dict(max_content_width=100, ignore_unknown_options=True, allow_extra_args=True)
+)
+@click.pass_context
+def pull_request(ctx: click.Context) -> None:
+    """Create pull request for the changes in each module."""
```

Comment:
I think this documentation needs to be expanded some. It should explain that PRs will be created in each of the repositories and that comments will be added with links between them.

Also, since this appears to accept arbitrary arguments, it should state that in the message and explain how those arguments should are going to be used.